ports/stm32/boards/MIKROE_QUAIL/board_init.c

```diff
@@ -0,0 +1,11 @@
+#include "py/mphal.h"
+
+void board_early_init(void) {
+
+#if !MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE
+    // set external SPI flash CS pin high
+    mp_hal_pin_output(MICROPY_HW_SPIFLASH_CS);
+    mp_hal_pin_write(MICROPY_HW_SPIFLASH_CS, 1);
```

Comment:
Why is this needed?  The CS pin should be configured as output and set high by default in the SPI flash ioctl init code.

Reply:
I just put together what I found in other ports without diving too much into the inner gears. This one comes from Sparkfun Micromod's [board_init.c](https://github.com/micropython/micropython/blob/aecb697c72ac1fdb9f549daa5734e2ffbd37a6d3/ports/stm32/boards/SPARKFUN_MICROMOD_STM32/board_init.c).

When removed, I can still see the filesystem on the external FLASH. Hence, yes, it's not required.